de_DE/LC_MESSAGES/discordbot.po

```diff
@@ -49,47 +49,47 @@ msgstr "%d Suchergebnisse:"
 
 # SongName by BandName
 msgid "MUSICMATCH_TITLE_BY_ARTIST"
-msgstr "%s von %s"
+msgstr "%s - %s"
 
 msgid "PATRON_TICKET_SUCCESS"
-msgstr "You received a %s Sudomemo Patron Ticket!"
+msgstr "Du hast ein %s Sudomemo Patron Ticket erhalten."
 
 msgid "REJECT_ACCOUNT_ALREADY_LINKED"
-msgstr "Du hast dein Sudomemo-Konto bereits verbunden!"
+msgstr "Du bist bereits in Sudomemo angemeldet."
```

Comment:
Again this gives off the impression of linking Discord accounts being logging in, when it is not.